package-lock.json

```diff
@@ -10089,11 +10089,6 @@
           "resolved": "https://registry.npmjs.org/prelude-ls/-/prelude-ls-1.2.1.tgz",
           "integrity": "sha512-vkcDPrRZo1QZLbn5RLGPpg/WmIQ65qoWWhcGKf/b5eplkkarX0m9z8ppCat4mlOqUsWpyNuYgO3VRyrYHSzX5g=="
         },
-        "prettier": {
```

Comment:
I see this gets bounced around a lot between PRs. While we generally prefer to avoid committing changes to the lock file when not required, it's not a great experience. It would be great to implement a change that would prevent this from happening in the first place. I'm not sure there's much we can do about it (except maybe switch to using Yarn or some other npm client instead?). Thoughts?